src/daria/image/image.py

```diff
@@ -112,25 +147,21 @@ def __init__(
                 "Invalid image data. Provide either a path to an image or an image array."
             )
 
-        # Read metadata from file or create from input arguments
-        if read_metadata_from_file:
-            self.create_metadata_from_file(metadata_path)
-        else:
-            self.origo = origo if isinstance(origo, np.ndarray) else np.array(origo)
-            self.width = width
-            self.height = height
-            self.dim = dim
-        self.shape = self.img.shape
+        if curvature_correction is not None:
+            self.img = curvature_correction(self.img)
+            self.width = curvature_correction.config["crop"]["width"]
+            self.height = curvature_correction.config["crop"]["height"]
+            self.origo = [0, 0]
```

Comment:
Do we want to overwrite `origo`? How is the procedure, if I would want to initialize an `daria.Image`, apply correction, and choose as `origo` something else then `[0,0]`? My suggestion is that the `metadata` is only updated in lines 151-155 `if metadata is None`. This would mean, `daria.Image` assumes that the provided `metadata` at initialization of a `daria.Image` is meaningful. Maybe even better, one could double check whether `width` and `height` in the `metadata` are identical to those in `curvature_correction`, and if not, a warning (maybe not an error) is thrown. And we would have to decide one convention, who is stronger: `metadata` or `curvature_correction`.